src/hooks/useFixedToBottomOnIos.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect } from 'react'
+
+import {
+  applyFixToElement,
+  handleMutation,
+  setupEventListeners,
+  tearDownEventListeners,
+  TOOLBAR_SELECTOR
+} from 'lib/patches/fixedToBottomIos'
+
+/**
+ * Custom hook that fixes an element to the bottom of the screen on iOS devices.
+ * Listens for the specified element and ensures it's fixed to the viewport bottom.
+ * Only activated on iOS devices.
+ */
+export const useFixedToBottomOnIOS = (): void => {
+  useEffect(() => {
+    // Only proceed if on iOS devices
+    if (!/iPhone|iPad|iPod/.test(window.navigator.userAgent)) return
+
+    let observer: MutationObserver | undefined
+
+    // Check if the element is already present and set up positions, in practice it's unlikely to be present on initial render
+    // If not present, set up a mutation observer to handle future additions
+    const existingElement = document.querySelector<HTMLElement>(
+      TOOLBAR_SELECTOR
+    )
+
+    if (existingElement) {
+      applyFixToElement(existingElement)
+      setupEventListeners(existingElement)
+    } else {
+      observer = new MutationObserver(handleMutation)
+      observer.observe(document.body, { childList: true, subtree: true })
```

Comment:
we could use the element react ref, but that would mean digging into atlaskit. Here I chose to not do that. But that would be possible. It was mostly an arbitrary decision

Reply:
I find that it makes it more complicated, given that we're moving away from React, but I understand that you want to get as far away from Atlaskit as possible